layouts/partials/posts/cusdis.html

```diff
@@ -0,0 +1,16 @@
+{{- if isset .Site.Params "cusdis" -}}
+    {{- if and (isset .Site.Params.cusdis "data_app_id") (not (eq .Site.Params.cusdis.repo "" )) (eq (.Params.disableComments | default false) false) -}}
```

Comment:
```suggestion
    {{- if and (isset .Site.Params.cusdis "data_app_id") (eq (.Params.disableComments | default false) false) -}}
```
Just removing an unused check. :)